flow360/component/simulation/models/surface_models.py

```diff
@@ -42,63 +42,94 @@ class BoundaryBase(Flow360BaseModel, metaclass=ABCMeta):
 class BoundaryBaseWithTurbulenceQuantities(BoundaryBase, metaclass=ABCMeta):
     """Boundary base with turbulence quantities"""
 
-    turbulence_quantities: Optional[TurbulenceQuantitiesType] = pd.Field(None)
+    turbulence_quantities: Optional[TurbulenceQuantitiesType] = pd.Field(
+        None,
+        description="The turbulence related quantities definition."
+        + "See :func:`TurbulenceQuantities` documentation.",
+    )
 
 
 class HeatFlux(SingleAttributeModel):
-    """Heat flux"""
+    """
+    :class:`HeatFlux` class to specify the heat flux for `Wall` boundary condition
```

Comment:
Why is there a link in the doc under every class in this file? It just links to the same page.

Reply:
because it looks good with the link style (the underline). No actually because most of the classes are already like this when I reviewed the PR and I thought it is too much work to change.

we can change this if requested.